bin/circleci/setup-integration-tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -ex
+
+# TODO: Update this occasionally from here:
```

Comment:
Should we set up a recurring task somewhere to ensure this stays fresh? Maybe as part of an end-of-sprint checklist?